src/wallet/rpcwallet.cpp

```diff
@@ -2386,8 +2387,10 @@ UniValue encryptwallet(const JSONRPCRequest& request)
     // BDB seems to have a bad habit of writing old data into
     // slack space in .dat files; that is bad if the old data is
     // unencrypted private keys. So:
-    StartShutdown();
```

Comment:
Above on line 2347 it says `"Note that this will shutdown the server.\n"` which should be removed now

Reply:
Done